### PR TITLE
Use permanent links in the primer comment

### PR DESCRIPTION
**Pull request by DanielNoord**
- [x] Add yourself to CONTRIBUTORS if you are a new contributor.
- [x] Write a good description on what the PR does.

## Type of Changes

|     | Type                   |
| --- | ---------------------- |
| ✓   | :hammer: Refactoring   |

## Description

Suggestion by @cdce8p. Instead of using the branch name that we prime on we use the commit that we prime on. This should make the links permanent following the normal Github scheme.

### CODE REVIEW

**Comment by jacobtylerwalls** [comment]
Forgive me @DanielNoord if this is obvious, but this PR will need to adjust the *.json test fixtures in order to pass the tests.

**Comment by DanielNoord** [comment]
Oops. I didn't know we tested this 😅

**Comment by DanielNoord** [comment]
Turns out I actually also forgot to fix something. Thanks to the tests 😄

File: tests/testutils/_primer/test_primer.py [hunk 1]
```diff
@@ -66,7 +60,7 @@ def test_compare(self, directory: Path) -> None:
 
     def test_truncated_compare(self) -> None:
         """Test for the truncation of comments that are too long."""
-        max_comment_length = 500
+        max_comment_length = 525
```
Comment (DanielNoord) [inline tests/testutils/_primer/test_primer.py:63]: Changed as I didn't want to deal with `<details>` getting cut off 😅

File: tests/testutils/_primer/test_primer.py [hunk 2]
```diff
@@ -36,12 +36,6 @@ def test_primer_launch_bad_args(args: list[str], capsys: CaptureFixture) -> None
     assert "usage: Pylint Primer" in err
 
 
-@pytest.mark.skipif(
```
Comment (DanielNoord) [inline tests/testutils/_primer/test_primer.py:39]: This is why I didn't see it originally. These tests take 2 seconds, let's just run them on all platforms.

**Review by jacobtylerwalls — APPROVED** [review]

**Comment by jacobtylerwalls** [comment]
Windows failure?

**Comment by DanielNoord** [comment]
Was able to fix the issue with `Windows` by using a `PurePosixPath` 🎉

**Review by Pierre-Sassoulas — APPROVED** [review]

**Pull request merged**